STABLE_RELEASES.md

```diff
@@ -0,0 +1,147 @@
+
+Earning this privilege should be considered to be no small feat and is by no
+means guaranteed by any quantifiable metric. It is a symbol of great trust of
+the community of this project.
+
+# 0.37
+
+The `0.37.x` release series will continue receiving bugfixes until the Cosmos Hub
+migrates to a newer release of the Cosmos-SDK.
+
+# 0.38
+
+Discontinued since **0.39.0** is published.
+
+# 0.39 - The Launchpad Release Series
+
+The **0.39 Launchpad** release series will be supported until 6 months after **0.40.0** is published. A fairly strict **bugfix-only** rule applies to pull requests that are requested to be included into a stable point-release.
+
+## What pull requests will be included in stable point-releases
+
+Pull requests that fix bugs that fall in the following categories do not require a **Stable Release Exception** to be granted to be included in a stable point-release:
+
+ * **Severe regressions**.
+ * Bugs that may cause **client applications** to be **largely unusable**.
+ * Bugs that may cause **state corruption or data loss**.
+ * Bugs that may directly or indirectly cause a **security vulnerability**.
+
+## What pull requests will NOT be automatically included in stable point-releases
+
+As rule of thumb, the following changes will **NOT** be automatically accepted into stable point-releases:
+
+ * **State machine changes**.
+ * **Client application's code-breaking changes**, i.e. changes that prevent client applications to build without modifications to the client's source code.
+ 
+ In some circumstances, PRs that don't meet the aforementioned criteria might be raised and asked to be granted a *Stable Release Exception*.
+ 
+ ## Stable Release Exception - Procedure
+ 
+1. Check that the bug is either fixed or not reproducible in `master`. It is, in general, not appropriate to release bugfixes for stable releases without first testing them in `master`. Please apply the label [0.39 (Launchpad)](https://github.com/cosmos/cosmos-sdk/labels/0.39%20LTS%20%28Launchpad%29) to the issue.
+2. Add a comment to the issue and ensure it contains the following information (see the bug template below):
+ * **[Impact]** An explanation of the bug on users and justification for backporting the fix to the stable release.
+ * A **[Test Case]** section containing detailed instructions on how to reproduce the bug.
+ * A **[Regression Potential]** section with a clear assessment on how regressions are most likely to manifest as a result of the pull request that aims to fix the bug in the target stable release.
+3. **Stable Release Managers** will review and discuss the PR. Once consensus surrounding the rationale has been reached and the technical review has successfully concluded, the pull request will be merged in the respective `release/launchpad/0.39.X` branch, being `X` the Launchpad's next point-release and the PR included in the `0.39.X` respective milestone.
+
+### Stable Release Exception - Bug template
+
```

Comment:
This should be a github issue template. putting it here makes it harder to follow a template

Reply:
That'd be convenient. The thing is that this template is meant to be used mainly in comments on bugs that already exist and have been fixed in master that someone wants to get fixed in stable releases

Reply:
```
2. Add a comment to the issue and ensure it contains the following information (see the bug template below):
```

Reply:
Maybe good to just add this to the bug template in a way that can be easily excluded if it's not a stable release exception?

Reply:
That sounds doable to me, yes.